### DISCUSSION TITLE
Github hackathon action.yml file not found

### DISCUSSION THREAD

**Discussion by alexmarginean16** [community/community#26170] [Actions]
<p>Hi,</p>
<p>Whenever I try to submit my repo as a new entry to Github Hackathon I get the error <strong>No action.yml file fonund</strong> but I have an <strong>action.yml</strong> file. Can anyone help me with this? </p>
<p><a href="https://github.com/alexmarginean16/Python-action" rel="nofollow noopener">This</a> is my repo.</p>
<p>Thanks!</p>

**Comment by bdougie** [comment, marked as answer]
<p>Hi, the hackathon built on top of Actions which is handling the submission checking. It is looking at your project’s root for the action.yml, but I see it in a folder. </p>
<p>I can review the code to see if this could be a tad more flexible, but the quick resolution is moving the action.yml file into the root and out of the action folder.  </p>

**Comment by alexmarginean16** [comment]
<p>Thank you so much, I did just that and I removed the actions folder, then I tweaked a bit the .github/workflows/action.yml file and I was able to submit the repo.</p>
